### PR TITLE
RELATED: RAIL-3439 unify version statuses

### PR DESCRIPTION
**Pull request by no23reason**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/733/ : SUCCESS in 1m 21s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/467/ : SUCCESS in 7s

File: docs/01_supported_versions.md [hunk 1]
```diff
@@ -35,9 +35,9 @@ The following table provides the lifecycle phases of GoodData.UI versions:
...
 | Major Version | Status | GA | EOD | EOS | EOL | Last Minor Version |
 | :--- | :--- | :--- | :--- | :--- | :--- | :--- |
-| 8 | Development | October 8, 2020 | _Not yet defined_ | _Not yet defined_ | _Not yet defined_ | 8.3 |
-| 7 | Maintenance | May 21, 2019 | October 8, 2020 | March 31, 2022 | _Not yet defined_ | 7.9 |
-| 6 | Not supported | November 1, 2018 | May 21, 2019 | October 8, 2020 | _Not yet defined_ | 6.3 |
-| 5 | Not supported | April 26, 2018 | November 1, 2018 | May 21, 2019 | _Not yet defined_ | 5.3 |
+| 8 | Generally available | October 8, 2020 | _Not yet defined_ | _Not yet defined_ | _Not yet defined_ | 8.3 |
+| 7 | End of Development | May 21, 2019 | October 8, 2020 | March 31, 2022 | _Not yet defined_ | 7.9 |
+| 6 | End of Development | November 1, 2018 | May 21, 2019 | October 8, 2020 | _Not yet defined_ | 6.3 |
+| 5 | End of Development | April 26, 2018 | November 1, 2018 | May 21, 2019 | _Not yet defined_ | 5.3 |
```
Comment (lupko) [inline docs/01_supported_versions.md:41]: according to ticket this should be End of Support?

Comment (no23reason) [reply docs/01_supported_versions.md]: 🤦  good catch, fixed

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/734/ : SUCCESS in 1m 10s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/468/ : SUCCESS in 7s

**Review by lupko — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-gate-PR419

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/735/ : SUCCESS in 1m 24s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/116/ : SUCCESS in 39s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sonarqube-scan-zuul-gate/18/ : SUCCESS in 42s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/207/ : SUCCESS in 22s

**Pull request merged**